PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -194,4 +194,50 @@ struct HfEventSelection : o2::framework::ConfigurableGroup {
 };
 } // namespace o2::hf_evsel
 
+namespace o2::hf_evsel_mc
+{
+// generated particles rejection types
+enum McCollisionRejection {
+  None = 0,
+  TimeFrameBorderCut,
+  ITSROFrameBorderCut,
+  PositionZ,
+  NMcCollisionRejection
+};
+
+struct HfEventSelectionMc {
+  // event selection parameters (in chronological order of application)
+  bool useSel8Trigger{false};       // Apply the ITS RO frame border cut
```

Comment:
`sel8 != ITS ROF border cut`. Since in line 226 you check only the ITS ROF border cut, call this `bool` in another way, e.g. `useITSROFBorderCut`